CustomDeserializers/Protobuf/readme.md

```diff
@@ -0,0 +1 @@
+This is a sample implementation of protobuf deserializer. To learn how to try out these samples, please follow the [Visual Studio Walkthrough](https://aka.ms/asadeserializer).
```

Comment:
Please add .proto file that was used for generating `MessageBodyProto.cs` file. That file also has command to generate the cs file, please mention it as well in the readme.md

Reply:
Please mentioned that SimulatedTemperatureEvents.protobuf is an example file and say how it was generated, including some stats about how many records are there and what kind of records are there.